src/composables/useTheme.ts

```
@@ -3,96 +3,230 @@ import vitesseLight from 'shiki/themes/vitesse-light.mjs'
+  theme: {
+    highlighterTheme: {
+      light: vitesseLight,
+      dark: vitesseDark,
+    },
+  },
+  request: {
+    defaultView: ref<'schema' | 'contentType'>('contentType'),
+    showBaseURL: ref<boolean>(false),
+  },
+  jsonViewer: {
+    deep: ref<number>(Infinity),
```

Biome

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)